SOURCES/xapi-1.249.3-update-xapi-conf.XCP-ng.patch

```diff
@@ -10,13 +10,20 @@ Index: xapi-1.249.3/scripts/xapi.conf
 +
```

Comment:
As the patch is modified, it should be renamed with the version of XAPI it is made for.